postcss.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  map: {
+    inline: false
```

Comment:
Unlike the **Design System** which uses `gulp-sourcemaps` we do need to specific a map config here when building the individual `./packages/*` files via _postcss-cli_.

(Compare with `./design-system/postcss.config.js`)